Find-and-Replace/Replace-text-with-data-from-database/.NET-Standard/Replace-text-with-data-from-database/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.Data;
+using System.Data.SqlClient;
+using System.IO;
+
+namespace Replace_text_with_data_from_database
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Set datasource and database values.
+                string datasource = "";
+                string database = "";
+                //Load the file stream into a Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Automatic))
+                {
+                    SqlConnection sqlConnection = new SqlConnection("Data Source=" + datasource + ";Initial Catalog=" + database + ";Integrated Security=True");
```

Comment:
//Create sql connection to get data from database

Reply:
@MohanaselvamJothi -I have added the comment